bibserver/dao.py

```diff
@@ -159,15 +158,15 @@ def bulk_upsert(cls, dataset, state=None):
             else:
                 id_ = make_id(data)
                 data['_id'] = id_
-            
+
             if '_created' not in data:
                 data['_created'] = datetime.now().strftime("%Y%m%d%H%M%S")
             data['_last_modified'] = datetime.now().strftime("%Y%m%d%H%M%S")
-            
+
             index_result = conn.index(data, db, cls.__type__, urllib.quote_plus(id_), bulk=True)
         # refresh required after bulk index
-        conn.refresh()
-    
+        # conn.refresh()
```

Comment:
Was this removed on purpose?  If so, delete it and the misleading comment above.

If it's an independent change, move it to a different commit/PR.